index.html

```diff
@@ -1,1642 +1,222 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>MiniApp Common UI Components</title>
-    <!-- <link rel="stylesheet" href="./local.css" /> -->
-
-    <style>
-    .two-cols {
-      display: grid;
-      grid-template-columns: 1fr 1fr;
-    }
-
-    table.members {
-      border: 1px #f0f0f0 solid;
-      margin-bottom: 1em;
-      width: 100%;
-    }
-    table.members th:nth-child(1),
-    table.members td:nth-child(1) {
-      width: 11rem;  
-    }
+<html lang="en">
 
-    table.members th:nth-child(2),
-    table.members td:nth-child(2) {
-      width: 5rem;  
-    }
-    table.members thead tr,
-    table.members tr:nth-child(even) {
-      background-color: #f0f0f0;
-    }
-    td, th {
-      padding:2px 15px;
-    }
-    </style>  
-  <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+<head>
+  <meta charset="utf-8" />
+  <title>MiniApp Components</title>
+  <script async class="remove" src="https://www.w3.org/Tools/respec/respec-w3c"></script>

```

Comment:
ReSpec docs recommend `defer` instead of `async`: https://respec.org/docs/#getting-started

Reply:
Thanks for the heads-up @xfq . 
Since this is split in some branches, I'll open a issue to clean it after the merge. I'll change it also in the rest of the specs.

Reply:
Opened issue #11

Reply:
Thank you!